client/executor/common/src/error.rs

```diff
@@ -31,12 +31,12 @@ pub enum Error {
 	#[error("Unserializable data encountered")]
 	InvalidData(#[from] sp_serializer::Error),
 
-	#[error(transparent)]
-	Trap(#[from] wasmi::Trap),
-
 	#[error(transparent)]
 	Wasmi(#[from] wasmi::Error),
 
+	#[error("Sandbox backend error: {0}")]
+	SandboxBackend(String),
```

Comment:
I think we should drop the `Backend` suffix. What do you think?

```suggestion
	#[error("Sandbox error: {0}")]
	Sandbox(String),
```